ts/features/bonus/cgn/store/reducers/activation.ts

```diff
@@ -5,6 +5,7 @@ import { cgnActivationStatus } from "../actions/activation";
 
 export enum CgnActivationProgressEnum {
   "UNDEFINED" = "UNDEFINED",
+  "TIMEOUT" = "TIMEOUT", // number of polling exceeded
   "PROGRESS" = "PROGRESS", // The request is started
   "PENDING" = "PENDING", // Polling time exceeded
   "ERROR" = "ERROR", // The request is started
```

Comment:
Does the request start having `ERROR` state?

Reply:
Obviously not! My bad a cp&p typo 😄 

Fixed in 8014a88